packages/adminrouter/docker/Dockerfile

```diff
@@ -8,9 +8,9 @@ ENV OPENRESTY_VERSION=1.13.6.1 \
     OPENRESTY_COMPILE_OPTS="" \
     VENV_DIR=/usr/local/venv \
     AR_BIN_DIR=/usr/local/adminrouter/ \
-    VTS_MODULE_DIR=/usr/local/src/nginx-vts-module-v0.1.18 \
-    VTS_MODULE_DOWNLOAD_SHASUM=8887748e0ef76e459aa2c86244baf382d4f7d4e5 \
-    VTS_MODULE_DOWNLOAD_URL=https://github.com/vozlt/nginx-module-vts/archive/v0.1.18.tar.gz \
+    VTS_MODULE_DIR=/usr/local/src/jongiddy-nginx-module-vts-0d39f4d \
+    VTS_MODULE_DOWNLOAD_SHASUM=79924d52764eefa8184e6e131124fda06d1c7256 \
+    VTS_MODULE_DOWNLOAD_URL=https://github.com/jongiddy/nginx-module-vts/tarball/escape \
```

Comment:
Should we fork the repo to the mesosphere org instead?

Reply:
This did cross my mind. I didn't do it because this is a patch to a package that we use as a third-party dependency (as opposed to a fork of a third-party package). The best outcome here is that the patch gets merged into the original repo and we change it to point to the next release.

The code compares the download to a hash, so we can detect if there are any changes to the branch.